variants/frontend-base/js-lint/template.rb

```diff
@@ -13,13 +14,22 @@
   "not IE 11",
   "not IE_Mob 11"
 ]
+package_json["jest"] = {
```

Comment:
I'd prefer to use `jest.config.js`, as I think it makes it easier to find (especially when you're not as familiar with the stack, since its not as obvious to check `package.json` for config).

This'll also make things more consistent with other projects that'll have more settings, and allows us to use type-checking to assist.